extensions/machine-learning-services/src/packageManagement/SqPackageManageProviderBase.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { ApiWrapper } from '../common/apiWrapper';
+import * as constants from '../common/constants';
+
+export class SqlPackageManageProviderBase {
```

Comment:
The file name here is missing an L in Sql.

Also nit - but shouldn't these be SqlPackageManagerProvider (Manage -> Manager)